### ISSUE TITLE
Can't close (inactive) channel without --force

### ISSUE THREAD

**Issue by stevenroose** [lightningnetwork/lnd#1032]
### Your environment

* version of `lnd`: 6fa93a78c18180a3bc77116b749b3c5e279d5a33
* which operating system (`uname -a` on *Nix): Arch recent update
* version of `btcd`, `bitcoind`, or other backend: btcd recent master

I had some inactive channels I wanted to get closed, but closing them without using `--force` gave the error `channel with chan_id=... not found`, while the channel could be closed when using `--force`.

Now, the peer of that channel was offline, so it might have been the case that the *peer* could not be found or that the channel could not be found *among the active channels*. In either case, the error message could be more helpful.

```
$ lncli closechannel --output_index 0 --funding_txid 40602af07ed2ae6448b9c3f0f4264843dd0f28d1065687eb06daa39aa120e7ad                                                                                                 
[lncli] rpc error: code = Unknown desc = channel with chan_id=ade720a19aa3da06eb875606d1280fdd434826f4f0c3b94864aed27ef02a6040 not found
```

Right afterwards (not the `--force` in the end):
```
$ lncli closechannel --output_index 0 --funding_txid 40602af07ed2ae6448b9c3f0f4264843dd0f28d1065687eb06daa39aa120e7ad --force
{
        "closing_txid": "5c6e4538ccbdeceb2717a0eb9a2b2198867c6b3f1f9a7bb68118d66ee1d1f7d1"
}
```

**Comment by Roasbeef** [comment]
Yes, you can't close out a channel normally if the remote peer isn't online, instead you have to force close it.

**Comment by stevenroose** [comment]
I realized, the error message suggests that the channel is not found, though (because the htlcswitch can't find it and the htlcswitch error is propagated up).
